source/admin_manual/migrating_mailboxes.rst

```diff
@@ -0,0 +1,200 @@
+.. _migrating_mailboxes:
+
+===================
+Migrating mailboxes
+===================
+
+General
+=======
+
+When doing mailbox migration between two systems, it is useful to first make a trimmed configuration file containing only the configuration you need for migration.
+
+You should use migration when you are changing aspects, such as compression, encryption or mail location driver; or want to restructure your mails in some way.
+
+Things that should be in your migration config:
+
+ * Mail user, home and location settings
+ * IMAP client settings (when needed)
+ * Namespace definitions, including public and shared namespaces.
+
+     * But no auto=create or auto=subscribe!
```

Comment:
Because those break sync, known fact. If you have

```
namespace {
   prefix = 
   mailbox Sent {
      auto = subscribe
  }
}
```

Sent will fail sync -1 always.

Reply:
Really? How exactly? Sounds like a bug. But should at least briefly say this here.

Reply:
I'll write something here, it's more complicated than it seems.